tests/call/test_file_reader_decorator.py

```diff
@@ -7,22 +7,30 @@
 
 import os
 import pytest
+import shutil
+import tempfile
 
 import fuzzinator
 
 from common_call import blinesep, resources_dir, MockAlwaysFailCall, MockNeverFailCall
 
 
-@pytest.mark.parametrize('call_init_kwargs, call_kwargs', [
-    ({'init_foo': 'init_bar'}, {'test': os.path.join(resources_dir, 'mock_tests', 'bar.txt')})
+@pytest.mark.parametrize('call_init_kwargs, call_kwargs, decorator_init_kwargs', [
+    ({'init_foo': 'init_bar'}, {'test': os.path.join(resources_dir, 'mock_tests', 'bar.txt')}, {'cleanup': False}),
+    ({'init_foo': 'init_bar'}, {'test': os.path.join(resources_dir, 'mock_tests', 'bar.txt')}, {'cleanup': True})
```

Comment:
Don't have to duplicate the entries here only to have cleanup with True and False. A new decorator can be stacked on top of the existing one and then pytest will create all the combinations. I.e.:

```py
@pytest.mark.parametrize('dec_kwargs', [
    {},
    {'cleanup': False},
    {'cleanup': True},
])
@pytest.mark.parametrize('call_init_kwargs, call_kwargs', [
    ({'init_foo': 'init_bar'}, {'test': os.path.join(resources_dir, 'mock_tests', 'bar.txt')})
])
```

Notes:
- the default of `cleanup` should also be tested, thus the empty dict
- `dec_kwargs` is the name used thoughout the tests